src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -34,8 +37,9 @@ public AddCommand parse(String args) throws ParseException {
         CompanyName companyName = ParserUtil.parseCompanyName(
                                                     argMultimap.getValue(PREFIX_COMPANY_NAME).orElse(null));
         JobTitle jobTitle = ParserUtil.parseJobTitle(argMultimap.getValue(PREFIX_JOB_TITLE).orElse(null));
+        Set<Review> reviewList = ParserUtil.parseReviews(argMultimap.getAllValues(PREFIX_REVIEW));
```

Comment:
I think we can create A Reviews class to store all reviews, which may be handy for enhancing GUI later